lms/djangoapps/gradebook/management/commands/tests/test_generate_gradebook_entries.py

```diff
@@ -133,6 +139,7 @@ def get_module_for_user(self, user, course, problem):
             course.id
         )._xmodule
 
+    @skip
```

Comment:
I'm not sure. I don't remember putting this skip in here. I'll remove it and see what happens!